core/common/src/thrift/file_system_master.thrift

```diff
@@ -115,6 +115,7 @@ struct TAclEntry {
   1: optional TAclEntryType type
   2: optional string subject
   3: optional list<TAclAction> actions
+  4: optional bool defaultEntry;
```

Comment:
NIT: I think I prefer `isDefault`, like the others.

Reply:
done